module/cmd-find/src/main/java/net/md_5/bungee/module/cmd/find/CommandFind.java

```diff
@@ -22,6 +23,10 @@ public void execute(CommandSender sender, String[] args)
         } else
         {
             ProxiedPlayer player = ProxyServer.getInstance().getPlayer( args[0] );
+            if ( player == null )
+            {
+                player = ProxyServer.getInstance().getPlayer( UUID.fromString( args[0] ) );
```

Comment:
A helper method for UUID.toString which returns null instead of throwing an exception when the string is no correct uuid would be better.